README.md

```diff
@@ -71,6 +80,81 @@ message Transaction {
 }
 ```
 
+### Generating Code
```

Comment:
This feels a longer-winded than necessary. I definitely wouldn't suggest people experiment with both buf and protoc 😛 

Instead of a whole section about this, just make it clear above that to _use_ the runtime library after `pip install`, they also need to generate the Python code for the core `buf.protovalidate` Protobuf package, and provide examples using `buf`. (I would leave `protoc` as an exercise for the reader: after all, very few people using `protoc` use it directly but usually have their own tooling around it or maybe use `bazel`.)

With `buf`, you can suggest that using `--include-imports` flag will do what they need or suggest they separately run `buf generate buf.build/bufbuild/protovalidate`.

Reply:
hows [62a6014](https://github.com/bufbuild/protovalidate-python/pull/54/commits/62a6014d42712605f6b1285a3040e383d67b6254)?

Reply:
Better. Ship it!!